### DISCUSSION TITLE
Error: Invalid version: "2.1"

### DISCUSSION THREAD

**Discussion by chris-nowicki** [prisma/prisma#23314] [Q&A] [answered]
### Question

I just installed prisma to do an introspective and start using with my planetscale database.

When I run `npx prisma db pull` I get the following error:

![CleanShot 2024-02-28 at 21 33 17@2x](https://github.com/prisma/prisma/assets/102450568/eb732649-b5a1-44f3-8282-0995a0e62a91)

not sure what is going on.  I have searched everywhere and can't find the issue.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "mysql"
  url      = env("DATABASE_URL")
}
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: macOS
- Database: PlanetScale
- Node.js version: 18.17.0

- Run `prisma -v` to see your Prisma version and paste it:

![CleanShot 2024-02-28 at 21 36 31@2x](https://github.com/prisma/prisma/assets/102450568/e191a8f3-b044-4f81-a056-c07135afd4b6)

**Comment by olup** [comment]
Have the same error with "invalid version V3". Latest Prisma with MySQL database (also the db pull never hits the db, because it will error the same even with a dummy url)

**Reply by chris-nowicki** [reply]
Hey @olup I found the problem.  What is the version version number in your package.json file?

**Reply by olup** [reply]
Not sure as I am afk. Was yours 2.1 ?

**Reply by olup** [reply, marked as answer]
`  "version": "v3",` to `  "version": "3.0.0",` in the package.json fixed it indeed. Too bad the error was not more detailed.
